pkg/controller/job/job_controller.go

```diff
@@ -770,6 +770,22 @@ func (jm *Controller) syncJob(ctx context.Context, key string) (forget bool, rEr
 		jm.queue.AddAfter(key, syncDuration)
 	}
 
+	klog.InfoS("______ syncJob check if to handle!!!!", "enabled", feature.DefaultFeatureGate.Enabled(features.JobBackoffPolicy), "BackoffPolicyOk", job.Spec.BackoffPolicy != nil, "restartPolicyOk", job.Spec.Template.Spec.RestartPolicy == v1.RestartPolicyNever, "finishedConditionOk", finishedCondition == nil)
```

Comment:
I suppose you don't plan to keep this line? :)

Reply:
yeah, it is a draft :)